docs/content/Examples-Tutorials-Recipes/Multiple-Data-Sources-With-Same-Schema.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Manage Access to Multiple Data Sources
+permalink: /recipes/multiple-sources-same-schema
+category: Examples & Tutorials
+subCategory: Data sources
+menuOrder: 3
+---
+
+## Use case
+
+We need to access the data from different data sources for different tenants.
+For example, we are the platform for the online website builder, and each client
+can only view their data. The same data schema is used for all clients.
+
+## Configuration
+
+Each client has own database. To manage access to the databases, we need to use
+the
+[`contextToAppId`](https://cube.dev/docs/config#options-reference-context-to-app-id)
+function and the
+[`driverFactory`](https://cube.dev/docs/config#options-reference-driver-factory)
+property. [JSON Web Token](https://cube.dev/docs/security) includes information
+about the tenant name in the `tenant` property.
+
+```javascript
+const PostgresDriver = require('@cubejs-backend/postgres-driver');
+
+module.exports = {
+  contextToAppId: ({ securityContext }) =>
+    `CUBEJS_APP_${securityContext.tenant}`,
+
+  driverFactory: ({ securityContext } = {}) => {
+    if (securityContext.tenant === 'cubeDev') {
+      return new PostgresDriver({
+        database: 'cubeDev',
```

Comment:
Do you think referencing Cube Dev here makes sense here? I suggest removing Cube Dev references and replacing them with something like "local database" and "remote database".

Reply:
It's a good idea. Rename to `localDB`

Reply:
Did you? I see no changes here in the diff, and via the Netlify preview link as well.

Reply:
It's my mistake, sorry :)